docker/docker_tools.py

```diff
@@ -317,31 +358,52 @@ def build(
                     print(
                         f"Dev mode: since {bks} doesn't exist, cloning the BookServer..."
                     )
-                    xqt("git clone https://github.com/bnmnetp/BookServer.git")
+                    try:
+                        # Check if possible to clone BookServer with Custom Repo
+                        xqt("export GIT_TERMINAL_PROMPT=0 && git clone https://github.com/%s/BookServer.git"%clone_bs)
```

Comment:
Same comment -- no `export`.

Reply:
Previously commented about ```export``` command being used for cloning.